specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/mfe.json

```diff
@@ -21156,6 +22197,228 @@
       },
       "additionalProperties": false
     },
+    "ServerlessEndpoint": {
+      "additionalProperties": false,
+      "properties": {
+        "authMode": {
```

Comment:
>authMode [](http://example.com/codeflow?start=9&length=8)

authenticationMode?

Reply:
our other, existing resources have this property named as "authMode so we'd like to keep consistent with that